### DISCUSSION TITLE
Get CHGeo2004 height instead of the ellipsoid

### DISCUSSION THREAD

**Discussion by pvocat** [opengisch/QField#3069] [Q&A]
Hello everyone,

First of all, sorry if I'm using the wrong words, I'm not a geoinformation professional, I'm the IT guy trying to set up an external GPS on QField 😓.
A colleague of mine needs to collect points in the field with QField, but he needs a more accurate GPS than the one built into his phone. We have a couple of Trimble R12i's that we were able to set up via bluetooth on the Trimble Mobile Manager app, and we are currently using the mock location feature to provide coordinates to QField.
Everything looks good, except for the height. The Trimble Mobile Manager app allows for two types of height measurements: ellipsoidal height and geoidal height (CHGeo2004).
Our current problem is that when we use the mock location, QField gets the coordinates from Trimble, but it only gets the ellipsoidal height, when we would need to get the CHGeo2004.
Do you know a way that would allow us to select the geoidal height, instead of the ellipsoid?

Thanks in advance to those who will take the time to answer me 🙏 !

**Comment by VxTedxV** [comment]
related to #2670 or #1601 ?

**Comment by pvocat** [comment]
Looks like it the same problem, but both of them used the grid shift method to correct it.
I didn't find it, I apparently didn't use the good keywords to look for it.
We'll try this tomorrow, I found a .geo file and a way to convert it in .gtx since I wasn't able to find the "chgeo04_htrans.gtx" showed on the discussion 1601. We'll see if it is indeed what we were looking for !
Thanks for your help.

**Reply by pvocat** [reply]
Everything seems to work as intended, thanks again and sorry if I wasn't able to find those two posts on my own !
